src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -452,7 +453,7 @@ const useGroupInfo = () => {
     getMembersList,
     handleOnNameChange,
     handleOnImageClicked,
-    uploadImageBase64,
+    uploadImage,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export of `uploadImage` has replaced `uploadImageBase64`. Ensure that all imports of this function have been updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->